src/components/Tutorial.vue

```diff
@@ -50,11 +51,11 @@ export default {
   components: {
     LessonLink
   },
-  data: self => {
+  data () {
```

Comment:
Could you help me understand why this one doesn't need the `self` and others instances do?

Reply:
in short, by using [`method definitions`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions) we can use `this.*` inside the `data` function.

but actually I'll revert this to be consistent with the current code base - although I think it's better to use `this.*`, `self` prop works just as fine.